scripts/fluentui-publish/jest.config.js

```diff
@@ -16,5 +16,7 @@ module.exports = {
     ],
   },
   coverageDirectory: './coverage',
+  testTimeout: 20000,
```

Comment:
we use integration test via calling createProjectGraph api, so test might take longer on CI